src/main/java/org/alliancegenome/mati/configuration/PostgresResource.java

```diff
@@ -13,26 +13,26 @@
 public class PostgresResource implements QuarkusTestResourceLifecycleManager, DevServicesContext.ContextAware {
 
     private Optional<String> matiContainerNetworkId;
-    private JdbcDatabaseContainer matiContainer;
-    private JdbcDatabaseContainer rolldownContainer;
-    private final String DOCKER_IMAGE = "postgres:13";
 
-    @Override
+	@Override
     public void setIntegrationTestContext(DevServicesContext context) {
         matiContainerNetworkId = context.containerNetworkId();
     }
 
     @Override
     public Map<String, String> start() {
-        matiContainer = new PostgreSQLContainer<>(DOCKER_IMAGE).withDatabaseName("mati").withLogConsumer(outputFrame -> {});
+		String DOCKER_IMAGE = "postgres:13";
```

Comment:
@abecerra at some point we probably could move to postgres:15

Reply:
Yeah, it would be a good upgrade